src/test/java/seedu/sudohr/logic/commands/department/EditDepartmentCommandTest.java

```diff
@@ -44,7 +44,7 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
         String expectedMessage = String.format(EditDepartmentCommand.MESSAGE_EDIT_DEPARTMENT_SUCCESS, editedDepartment);
 
         Model expectedModel = new ModelManager(new SudoHr(model.getSudoHr()), new UserPrefs());
-        expectedModel.setDepartment(model.getDepartment(DEPARTMENT_NAME_FIRST), editedDepartment);
+        expectedModel.removeDepartment(new Department(DEPARTMENT_NAME_SECOND));
```

Comment:
the line below this is intended? actually im not too sure of the logic behind this test case, help clarify thx!